Enable multiline comments in report previewer and focus on 'add card' comment box (#155)
insightsengineering/teal.reporter: NEWS.md

# teal.reporter 0.1.1.9000

### New features

> * When adding a card, the cursor is focused in the comment block.

Reviewer: ```suggestion
* Updated the comment block to have an active cursor when adding a card.
```

> * Report Previewer renders markdown as in downloaded document.

Reviewer: According to our good practice, news entries are preferably in the following format:
"past verb + statement", for example:

> * Report Previewer renders markdown as in downloaded document.

Reviewer: ```suggestion
* Updated report previewer to render markdown as in the downloaded document.
```

# teal.reporter 0.1.1.9000

### New features

> * Updated the comment block to have an active cursor when adding a card.

Reviewer 2: ```suggestion
* Updated the AddCardModule comment input to have an active cursor when adding a card.
```

> * Updated report previewer to allow multiline comment.

Reviewer 2: ```suggestion
* Updated report previewer to support preview of multiline comment.
```

> * Updated report previewer to render markdown as in the downloaded document.

Reviewer 3: ```suggestion
* Updated report previewer to allow multiline comment.
```
